### DISCUSSION TITLE
slash command

### DISCUSSION THREAD

**Discussion by abdiyaB** [jagrosh/MusicBot#1312] [Ideas]
will u be adding slash command ?

**Comment by bup0441** [comment]
there's no ETA for it but yes, slash commands are planned

**Reply by nooksmr** [reply]
Jesus, thanks you

**Reply by glappa** [reply]
i really would like slash commands, as using @ is very annoying when the bot is not the first account selected

**Reply by ghost** [reply]
I mean slash commands and all are nice but for your specific issue you could just change the prefix to something else. just put it to „!“ or something.

**Comment by glappa** [comment]
yea, but can you tell me where to change that in the source code?
thank you in advance

Am Mi., 15. März 2023 um 14:09 Uhr schrieb tonischumacher1337 <
***@***.***>:

**Comment by ghost** [comment]
You don‘t need to touch the code at all. It‘s a setting in the config file. You can also use the prefix command in discord chat to change it. Am 16.03.2023 um 20:10 schrieb Glappa ***@***.***>:﻿
yea, but can you tell me where to change that in the source code?
thank you in advance

Am Mi., 15. März 2023 um 14:09 Uhr schrieb tonischumacher1337 <
***@***.***>:

**Reply by glappa** [reply]
thank you for letting me know, it is really that easy!
